README.md

```diff
@@ -70,6 +72,16 @@ func validate() {
 }
 ```
 
+If instead you prefer to keep the information out of the Info.plist, you can use:
+``` swift
+let recaptcha = try? ReCaptcha(
+    apiKey: "YOUR_RECAPTCHA_KEY", 
+    baseURL: URL(string: "YOUR_RECAPTCHA_DOMAIN")!
+    )
```

Comment:
Could you fix this closing parenthesis so that it's in the beginning of the line, please?

Reply:
done :)